src/processor/html.ts

```diff
@@ -143,6 +179,115 @@ export function generateTocHtml({
   return prettier.format(toHTML(toc), { parser: 'html' });
 }
 
+const getCoverHtmlStyle = ({
+  pageBreakBefore,
+}: {
+  pageBreakBefore?: 'recto' | 'verso' | 'left' | 'right';
+}) => /* css */ `
+:root {
+  --page-mbox-visibility: hidden;
+  ${pageBreakBefore ? `break-before: ${pageBreakBefore};` : ''}
+}
+body {
+  margin: 0;
+}
+[role="doc-cover"] {
+  display: block;
+  width: 100vw;
+  height: 100vh;
+  object-fit: cover;
+}
+@page {
+  margin: 0;
+  @top-left-corner {
+    visibility: var(--page-mbox-visibility);
+  }
+  @top-left {
+    visibility: var(--page-mbox-visibility);
+  }
+  @top-center {
+    visibility: var(--page-mbox-visibility);
+  }
+  @top-right {
+    visibility: var(--page-mbox-visibility);
+  }
+  @top-right-corner {
+    visibility: var(--page-mbox-visibility);
+  }
+  @left-top {
+    visibility: var(--page-mbox-visibility);
+  }
+  @left-middle {
+    visibility: var(--page-mbox-visibility);
+  }
+  @left-bottom {
+    visibility: var(--page-mbox-visibility);
+  }
+  @right-top {
+    visibility: var(--page-mbox-visibility);
+  }
+  @right-middle {
+    visibility: var(--page-mbox-visibility);
+  }
+  @right-bottom {
+    visibility: var(--page-mbox-visibility);
+  }
+  @bottom-left-corner {
+    visibility: var(--page-mbox-visibility);
+  }
+  @bottom-left {
+    visibility: var(--page-mbox-visibility);
+  }
+  @bottom-center {
+    visibility: var(--page-mbox-visibility);
+  }
+  @bottom-right {
+    visibility: var(--page-mbox-visibility);
+  }
+  @bottom-right-corner {
+    visibility: var(--page-mbox-visibility);
+  }
+}
```

Comment:
https://github.com/vivliostyle/vivliostyle-cli/commit/0fd016a850c18690fc6d1456ae50d8813410b678 "chore: Don't inherit root Themes for cover documents" により、ルートのテーマのためにカバーページにページマージンがつく問題と、ページマージンボックス内容が表示される問題はなくなりました。
これにより `--page-mbox-visibility: hidden;` と各マージンボックスの `visibility: var(--page-mbox-visibility);` も不要になったのではないでしょうか？
おそらく意図としては、ルートのテーマ以外のスタイルシートがある場合にカバーページのページマージンボックス内容をデフォルトで非表示にしたいということかと思います。しかし、そのスタイルシートでページマージンボックスが `visibility: visible` となる設定がある（`@vivliostyle/theme-base` をベースにしてればそうなる）ならば、うまくいきません。

カバーページHTMLに出力するスタイルシートはなるべくシンプルにしたほうが、ルートのテーマ以外のスタイルシートを追加してカスタマイズする場合にやりやすいと思います。

それからもう一点、`object-fit: cover` だとページサイズの縦横比が画像のと一致しないとき画像の上下あるいは左右が消えてしまいます。ページサイズ指定なしでウインドウサイズのページサイズで閲覧するような場合、ページと画像の縦横比は大きく違うことになり、画像の大事なところが見えなくなるということが起きがちです。 `object-fit: contain` のほうがよいと思います。

```suggestion
${
  pageBreakBefore
    ? `:root {
  break-before: ${pageBreakBefore};
}`
    : ''
}
body {
  margin: 0;
}
[role="doc-cover"] {
  display: block;
  width: 100vw;
  height: 100vh;
  object-fit: contain;
}
@page {
  margin: 0;
}
```